.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = false
```

Comment:
false on this?

Reply:
Not sure. It's one of the default options when you auto-generate an editorconfig

Reply:
I just set it to true since I can't imagine it ever getting in the way.